### PR TITLE
Added Igbo language.

### PR DESCRIPTION
**Pull request by PluckyPrecious**
Created the integration patch for the newly added Igbo language for 3.1 release.

### CODE REVIEW

**Comment by felixxm** [comment]
Adding a new language requires Transifex setup first. Do you intend to start a Igbo translation for Django and take the role of Igbo language coordinator? If you do, we can set it up on Transifex. Please give us your Transifex username. (\cc @claudep)

**Comment by claudep** [comment]
@felixxm, see https://groups.google.com/forum/#!topic/django-i18n/YzurStHQN5E

File: django/conf/locale/__init__.py [hunk 1]
```diff
@@ -270,12 +270,18 @@
...
+        'code': 'ig',
+        'name': 'Igbo',
+        'name_local': 'Asụsụ Ìgbò',
     },
     'id': {
         'bidi': False,
-        'code': 'id',
-        'name': 'Indonesian',
+        'codeg: 'id',
+        'namegb: 'Indonesian',
```
Comment (claudep) [inline django/conf/locale/__init__.py:284]: Obviously this is unwanted changes.

File: django/conf/locale/ig/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (claudep) [inline django/conf/locale/ig/__init__.py:1]: I think you should be able to create a completely empty `__init__.py`

**Comment by felixxm** [comment]
Thanks Claude.

**Comment by PluckyPrecious** [comment]
> Adding a new language requires Transifex setup first. Do you intend to start a Igbo translation for Django and take the role of Igbo language coordinator? If you do, we can set it up on Transifex. Please give us your Transifex username. (\cc @claudep)

Hi @felixxm i have done the Transifex setup and started translation already. Here's my username- PluckyPrecious, thanks.

**Comment by felixxm** [comment]
@PluckyPrecious Thanks :+1:

File: django/conf/locale/__init__.py (hunk 1 above)
Comment (PluckyPrecious) [reply django/conf/locale/__init__.py]: Yes it is, there was an error on my part during input which let to that. Fixed now.

**Comment by claudep** [comment]
It would be great if you could at least run `flake8` tests to catch basic errors before pushing your modifications. Tell us if you need help doing that.

File: django/conf/locale/ig/formats.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# This file is distributed under the same license as the Django package.
+#
+# The *_FORMAT strings use the Django date format syntax,
+# see https://docs.djangoproject.com/en/dev/ref/templates/builtins/#date
+DATE_FORMAT = 'j E Y г.'
+TIME_FORMAT = 'G:i'
+DATETIME_FORMAT = 'j E Y г. G:i'
+YEAR_MONTH_FORMAT = 'F Y г.'
```
Comment (felixxm) [inline django/conf/locale/ig/formats.py:8]: @PluckyPrecious Can you confirm using `'r.'` in formats?

**Comment by PluckyPrecious** [comment]
> @PluckyPrecious Thanks 👍 I squashed commits and fixed formatting.

Thanks a bunch @felixxm you rock.👍

File: django/conf/locale/ig/formats.py (hunk 3 above)
Comment (PluckyPrecious) [reply django/conf/locale/ig/formats.py:8]: @felixxm confirm what please i'm lost?

Comment (felixxm) [reply django/conf/locale/ig/formats.py:8]: Can you confirm that `DATE_FORMAT`, `DATETIME_FORMAT`, and `YEAR_MONTH_FORMAT` should contain the `г.' suffix for a year? I think it was copied from the `tg` format.

File: django/conf/locale/ig/formats.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    '%d.%m.%Y %H:%M:%S',     # '25.10.2006 14:30:59'
+    '%d.%m.%Y %H:%M:%S.%f',  # '25.10.2006 14:30:59.000200'
+    '%d.%m.%Y %H:%M',        # '25.10.2006 14:30'
+    '%d.%m.%Y',              # '25.10.2006'
+    '%d.%m.%y %H:%M:%S',     # '25.10.06 14:30:59'
+    '%d.%m.%y %H:%M:%S.%f',  # '25.10.06 14:30:59.000200'
+    '%d.%m.%y %H:%M',        # '25.10.06 14:30'
+    '%d.%m.%y',              # '25.10.06'
+]
+DECIMAL_SEPARATOR = ','
```
Comment (felixxm) [inline django/conf/locale/ig/formats.py:30]: @PluckyPrecious Can you confirm that `DECIMAL_SEPARATOR` is a comma? I found that it's probably a dot `'.'` :thinking:

Comment (PluckyPrecious) [reply django/conf/locale/ig/formats.py]: Hi @felixxm fixed.

**Comment by felixxm** [comment]
@PluckyPrecious Thanks :+1: Welcome aboard :boat:

**Comment by PluckyPrecious** [comment]
> @PluckyPrecious Thanks 👍 Welcome aboard ⛵

Thank you. 😅

**Pull request merged**